loadtest/beats-yaml-generator.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+# This command produces an output we parse to get the logstash node IPs, example here:
+# NAME                            READY   STATUS    RESTARTS   AGE   IP              NODE                                          NOMINATED NODE   READINESS GATES
+# logstash-pool-9b468dd96-x9g9c   1/1     Running   0          81s   192.168.16.90   ip-192-168-26-84.us-west-2.compute.internal   <none>           <none>
+with os.popen('kubectl get pods -o wide -n logstash') as stream:
+    pods_string = stream.read()
+pods_list = pods_string.split("\n")
+
+hosts_list = []
+i = 0
+for line in pods_list:
```

Comment:
Use a list slice to avoid having to need `i`
`for line in pod_list[1:]`

Reply:
I always forget about slicing